.travis.ubuntu.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e -x
+
+Xvfb :99 & export DISPLAY=:99
+
+./gradlew --version --console=plain
+./gradlew spotlessCheck check jacocoJunit5TestReport --stacktrace --console=plain -PlogTests -Pheadless
+
+bash <(curl -s https://codecov.io/bash)
```

Comment:
EOL at EOF

Reply:
You know what, you are absolutely right. I'm going to add a spotless rule to gradle about this.

Reply:
This should be forever resolved in this file because of this new spotless check:
https://github.com/wpilibsuite/shuffleboard/pull/220/files#diff-392475fdf2bc320d17762ed97109a121R42